x-pack/plugin/esql/qa/testFixtures/src/main/resources/inlinestats.csv-spec

```diff
@@ -34,31 +34,31 @@ FROM employees
 | SORT emp_no ASC
 | LIMIT 5;
 
-emp_no:integer | languages:integer | gender:keyword | max_lang:integer
```

Comment:
There is a series of changes to existent queries that already worked, where the order of the columns was, imo, wrong.
`inlinestats` should behave like `stats` where the aggregate comes first, the groupings come afterwards.

In this very specific case of this query where the `keep` is explicit about `gender` being the last column from left to right, `inlinestats` will put `max_lang` before `gender` because `gender` is its grouping field.